tests/util/core/test_match.py

```diff
@@ -156,3 +157,60 @@ def test_diff_track(self):
         assert track1.track_num != track2.track_num
 
         assert get_match_value(track1, track2) < self.LOW_MATCH_THRESHOLD
+
+    def test_duration_tolerance_thresholds(self):
+        """Test duration matching tolerance thresholds."""
+        album = MetaAlbum(title="Test Album", artist="Test Artist")
+        track1 = MetaTrack(album=album, title="Test Track", track_num=1, disc=1)
+        track2 = MetaTrack(album=album, title="Test Track", track_num=1, disc=1)
+
+        track1.duration = 180.0
+        track2.duration = 184.0
```

Comment:
This is a good test, but I think it would be more easily understandable if track 2's durations were set to a percentage of track 1's duration.That way you could see where you are on the scale instead of having to try and think if 184 is within 2.5% of 180.  That plus a few comments explaining the logic again such as "track 2's duration is > 10% from track 1 therefore the full match penalty is applied" or something.